pangea-sdk/v3/service/audit/api.go

```diff
@@ -963,3 +963,61 @@ type DownloadResult struct {
 func (a *audit) DownloadResults(ctx context.Context, input *DownloadRequest) (*pangea.PangeaResponse[DownloadResult], error) {
 	return request.DoPost(ctx, a.Client, "v1/download_results", input, &DownloadResult{})
 }
+
+// @summary Log streaming endpoint
+//
+// @description This API allows 3rd party vendors (like Auth0) to stream events
+// to this endpoint where the structure of the payload varies across different
+// vendors.
+//
+// @operationId audit_post_v1_log_stream
+//
+// @example
+//
+//	type LogStreamEventData struct {
```

Comment:
Does it make sense to have this struct if they should depend on each vendor? Are we sure that each vendor has this same structure?

Reply:
> Are we sure that each vendor has this same structure?

They don't, that is why the struct is not a part of the SDK. It is up to the user to define a struct for the data they're sending, and that's what's happening in this example snippet.